frontends/web/src/routes/account/send/send.tsx

```diff
@@ -628,201 +627,192 @@ class Send extends Component<Props, State> {
+              {
+                isSent && (
+                  <WaitDialog>
+                    <div className="flex flex-row flex-center flex-items-center">
+                      <Checked style={{ height: 18, marginRight: '1rem' }} />{t('send.success')}
+                    </div>
+                  </WaitDialog>
+                )
+              }
+              {
+                isAborted && (
+                  <WaitDialog>
+                    <div className="flex flex-row flex-center flex-items-center">
+                      <Cancel alt="Abort" style={{ height: 18, marginRight: '1rem' }} />{t('send.abort')}
+                    </div>
+                  </WaitDialog>
+                )
+              }
+              <Dialog
+                open={activeScanQR}
+                title={t('send.scanQR')}
+                onClose={this.toggleScanQR}>
+                {videoLoading && <Spinner guideExists />}
+                <video
+                  id="video"
+                  width={400}
+                  height={300 /* fix height to avoid ugly resize effect after open */}
+                  className={style.qrVideo}
+                  onLoadedData={this.handleVideoLoad} />
+                <div className={['buttons', 'flex', 'flex-row', 'flex-between'].join(' ')}>
+                  <Button
+                    secondary
+                    onClick={this.toggleScanQR}>
+                    {t('button.back')}
+                  </Button>
                 </div>
-              </WaitDialog>
-            )
-          }
-          <Dialog
-            open={activeScanQR}
-            title={t('send.scanQR')}
-            onClose={this.toggleScanQR}>
-            {videoLoading && <Spinner guideExists />}
-            <video
-              id="video"
-              width={400}
-              height={300 /* fix height to avoid ugly resize effect after open */}
-              className={style.qrVideo}
-              onLoadedData={this.handleVideoLoad} />
-            <div className={['buttons', 'flex', 'flex-row', 'flex-between'].join(' ')}>
-              <Button
-                secondary
-                onClick={this.toggleScanQR}>
-                {t('button.back')}
-              </Button>
-            </div>
-          </Dialog>
-        </div>
-        <Guide>
-          <Entry key="guide.send.whyFee" entry={t('guide.send.whyFee')} />
-          { isBitcoinBased(account.coinCode) && (
-            <Entry key="guide.send.priority" entry={t('guide.send.priority')} />
-          )}
-          { isBitcoinBased(account.coinCode) && (
-            <Entry key="guide.send.fee" entry={t('guide.send.fee')} />
-          )}
-          { isBitcoinOnly(account.coinCode) && (
-            <Entry key="guide.send.change" entry={t('guide.send.change')} />
-          )}
-          <Entry key="guide.send.revert" entry={t('guide.send.revert')} />
-          <Entry key="guide.send.plugout" entry={t('guide.send.plugout')} />
-        </Guide>
```

Comment:
guide missing?

Reply:
Yes, it's added in another commit here as we also make it into a different file. Commit: https://github.com/digitalbitbox/bitbox-wallet-app/pull/2190/commits/f24a3926b41bdf6bbdd5517e32daf572626d0ec0